src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Models/AsAzureClientSession.cs

```diff
@@ -28,7 +28,7 @@ namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane
     public class AsAzureClientSession
     {
         public const string RestartEndpointPathFormat = "/webapi/servers/{0}/restart?api-version=2016-10-01";
-        public const string LogfileEndpointPathFormat = "/webapi/servers/{0}/logfileHere";
+        public const string LogfileEndpointPathFormat = "/webapi/servers/{0}/logfile";
```

Comment:
@athipp Can you add a test for using the log file endpoint [please file an issue and reference it here if you can't do this right away]

Reply:
Also, please add a description of the bug fix to your changelog.md file